### PR TITLE
add users link to sidenav

### PR DESCRIPTION
**Pull request by jseppi**
This PR includes some of the changes from #1177 in order to add the "users" link to the SideNav. I wanted to get that link in before today's review/deploy so that folks would be able to navigate to the new Users page.

### CODE REVIEW

File: frontend/components/site/SideNav/sideNav.js [hunk 1]
```diff
@@ -25,17 +26,22 @@ class SideNav extends React.Component {
...
           />
           <SideNavItem
+            href={this.getUrl(siteId, sideNavPaths.USERS)}
+            icon="gear"
+            linkText={sideNavPaths.USERS}
+          />
+          <SideNavItem
             href={this.getUrl(siteId, sideNavPaths.PUBLISHED)}
-            icon='media'
+            icon="media"
```
Comment (wslack) [inline frontend/components/site/SideNav/sideNav.js]: why do we need double quotes and not single?

File: frontend/sass/_icons.scss [hunk 2]
```diff
@@ -97,3 +97,16 @@ $twelfth-column: 0;
...
+.icon.icon-gear {
+  &:before {
+    @include icon();
+    height: 1em;
+    background-position: $second-column $second-row;
+    content: 'Gear icon';
+  }
+
+  &:hover:before {
+    background-position-x: $fifth-column;
```
Comment (wslack) [inline frontend/sass/_icons.scss]: why is this needed?

**Comment by wslack** [comment]
@jseppi Jonathan is in a login.gov fire so I checked it. Two questions.

File: frontend/components/site/SideNav/sideNav.js (hunk 1 above)
Comment (jseppi) [reply frontend/components/site/SideNav/sideNav.js]: That's a linting thing. In JSX, we use double-quotes for attribute values because that is what our preferred config (`eslint-config-airbnb`) is set to use. Either way doesn't matter, but consistency is good so we just go with what the config specifies :)

File: frontend/sass/_icons.scss (hunk 2 above)
Comment (jseppi) [reply frontend/sass/_icons.scss]: that's for "swapping" out the black icon to the blue icon on mouse hover.

Comment (wslack) [reply frontend/sass/_icons.scss]: ah! clever.
